refinery/ui/source/js/commons/partials/sharing-help-popover-details.html

```diff
@@ -0,0 +1,15 @@
+<p>
+  Only data set owners can change sharing options. To share
+  with an authenticated user, create a new group or invite the user to an
+  existing group
+  <span ng-if="!$ctrl.isCollaborationPage">
+    <a href='/collaboration'> on the collaboration page. </a> Owners can
+    update data set permissions (including making it public) on the data set
+    preview by clicking the “sharing” button.
+  </span>
+  <span ng-if="$ctrl.isCollaborationPage">
+    on the collaboration page. Owners can update data set permissions
+    (including making it public) on the <a href='/#'> data set preview </a> by
```

Comment:
Could this be more confusing than useful? We can't actually take them to the data set preview, but if we just take them to the dashboard, there isn't going to be a "sharing" button for them to click either.

Reply:
@mccalluc Good point. How about this revised text: 
"Only data set owners can change sharing options. To share
  with an authenticated user, create a new group or invite the user to an
  existing group on the collaboration page. Owners can update data set permissions (including making it public) on the <a href=/#> dashboard </a> by opening a data set preview (click a data set link) and then clicking the “sharing” button."

Reply:
That sounds good.